discovery-infra/start_discovery.py

```diff
@@ -574,6 +575,46 @@ def execute_day1_flow(cluster_name):
             )
             cluster_deployment.wait_for_state("insufficient")
 
+        else:
+            cluster = client.create_cluster(
+                cluster_name,
+                ssh_public_key=args.ssh_key, **_cluster_create_params()
+            )
+
+        if args.with_static_network_config:
+            tf_folder = utils.get_tf_folder(cluster_name, args.namespace)
+            static_network_config = static_network.generate_static_network_data_from_tf(tf_folder)
+            if args.kube_api:
```

Comment:
Shouldn't it be verified on a higher level? Or is it only for static networking?

Reply:
required only for static networking